shell/src/main/java/alluxio/cli/fs/command/StatCommand.java

```diff
@@ -107,7 +122,7 @@ public String getUsage() {
 
   @Override
   public String getDescription() {
-    return "Displays info for the specified path both file and directory."
+    return "Displays info for the specified path or fileId both file and directory."
         + " Specify -f to display info in given format:"
         + "   \"%N\": name of the file;"
         + "   \"%z\": size of file in bytes;"
```

Comment:
This section can use more formatting:

```suggestion
    return String.join("\n", Arrays.asList(
        "Displays info for the specified file or directory.",
        "Specify -fileId to treat the first positional argument as a file ID.",
        "Specify -f with a template string to display info in the given format. These format placeholders are available:",
        "  %N: name of the file",
        "  %z: size of file in bytes",
        ...
    ));
```

Reply:
Adding a new format option `%i` to print the file Id if the user specifies the file with a path would be helpful. This way we can get a file's ID by its path.